src/main/msp/msp.c

```diff
@@ -1457,11 +1457,12 @@ case MSP_NAME:
 #endif
         break;
 
-#ifdef USE_MAG
     case MSP_COMPASS_CONFIG:
+#ifdef USE_MAG
         sbufWriteU16(dst, imuConfig()->mag_declination);
-        break;
 #endif
+        break;
```

Comment:
Do you need to write an empty value here if use_mag is not defined, like for the read?
eg
```
#else
    sbufWriteU16(dst, 0);
```

Reply:
Thanks. Added.